traveler-ios-ui/TravelerKitUI/ViewControllers/BookableDetailsViewController.swift

```diff
@@ -53,7 +53,7 @@ open class BookableDetailsViewController: UITableViewController {
         errorContext?.addObserver(self)
     }
 
-    deinit {
+    override open func viewWillDisappear(_ animated: Bool) {
         errorContext?.removeObserver(self)
```

Comment:
You shouldn't remove an observer in `viewWillDisappear` unless you're adding it in `viewWillAppear`.  Otherwise, the observer will be removed and never added again if the view goes out of sight and then back into sight.